src/package.json

```diff
@@ -67,13 +67,13 @@
     "@types/dragula": "^2.1.33"
   },
   "peerDependencies": {
```

Comment:
Should this file also be updated to match the downgraded d3 version?

Reply:
I'm not fully sure that downgrading is the correct solution now. I built the library with d3@5.16 and installed it into a CLI project. In production mode I did not see the rgb error. So it seems like it's something to do with our documentation app.

Also noticed that this is broken in our 2.0.6 release, so one option is to make another ticket and deal with it separately.

https://uxaspects.github.io/UXAspects/#/charts/donut-charts#nested-donut-chart

If you have any ideas, let me know. I'll look into it a bit further.

Reply:
Hmm, thats interesting! Perhaps we need to tweak our webpack optimization settings. I will take a look into it, thanks!

Reply:
Yes iot's probably something we are doing slightly differently in our webpack config. However I think I should revert the last commit here and deal with that at a later date, what do you think?

Reply:
Yes, I don't think we should downgrade d3. I'm making our config the same as the CLI's to see if that makes any difference, if not then yeah we should raise another ticket.

Reply:
That's a good idea, thanks for your help!

Reply:
I have just pushed some changes which should hopefully fix the issue 🤞

Reply:
Thanks! It looks good. I'll make the same changes in the other repo.

Reply:
Thank you!

Reply:
Unfortunately those changes do not work in the downstream repo. The site will not load in production. Where did you find the changes to make?

I'm still inclined to defer this to another sprint. We are losing a lot of QA time to a pre-existing issue now.

Reply:
I'm not sure of the exact cause, I believe it could be the version of the TerserPlugin, so you might be able to just update that. I just updated the config and dependencies to match those used in the CLI. I'm happy to defer it if its not trivial 👍!

Reply:
It has the same dependencies as far as I can see. However the webpack config there is significantly different from this one. I tried a couple of other changes but it's breaking in various ways so I'll leave it until we have some time to spend on it.

Reply:
Sounds good! Are you happy to keep the changes in this repo? Or would you like me to revert them?

Reply:
I think it's fine here. It should not impact the library or the internal documentation site as far as I'm aware. The downstream build is running so we can verify when that's complete.

Reply:
Seems like it's now working okay with the older dependencies, so we can probably leave it until we switch over to Angular CLI:

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/caf_CI_ux-aspects-micro-focus_EL-3986-angular-10/#/ui-components/charts#nested-donut-chart